src/main/js/EditorPage.jsx

```diff
@@ -378,7 +378,7 @@ class PipelineLoader extends React.Component {
                       "path": "Jenkinsfile",
                       branch: saveToBranch || this.defaultBranch,
                       sourceBranch: branch,
-                      repo: repo,
+                      repo: repo ? repo : team, // if no repo, this is not in an org folder
```

Comment:
But you don't need it, see my other comment as all you need is to fire off indexing using `RunApi.startRun()`.